File: C_programming/primeno.c
//12.wap to accept a no. and print whether it is a prime no. or not.

int main()
{
	int n,i,flag=0;
	printf(" enter a number :");
	scanf("%d",&n);
	 for(i=2;i<=n/2;i++)
	 {
	 	//condition for non prime.
	 	if(n%i == 0)
	 	{
	 		flag = 1;
	 		break;
		}
	 }
	 if(n == 1)
	 {
	 	printf("1 is neither prime nor composite.");
	 }
	 else{
	 	if (flag == 0)
	 	printf("%d is a prime no.",n);
	 	else
	 	printf("%d is not a prime no.",n);
	 	
	     }
	 
}

File: Java_Programs/Java_Programs/PrimeNo.java
//11.wap to accept a no. and then print whether it is prime no.or not.
import java.util.*;
class PrimeNo
{
  public static void main(String args[])
  {
	  int num,i=2;
	  boolean flag= false;
	Scanner s=new Scanner(System.in);
	System.out.println("enter a number :");
	num=s.nextInt();

	while(i<=num/2)
	{
		//condition for non prime
		if(num%i==0)
		{
			flag =true;
			break;
		}
		i++;
	}
	if(!flag)
	 System.out.println(num+ "is a prime number.");
	 else
	 System.out.println(num+ "is not a prime number.");
	}
}